ethereum/statetransition/src/test-support/java/tech/pegasys/artemis/statetransition/BeaconChainUtil.java

```diff
@@ -94,6 +112,16 @@ public BlockProcessingRecord createAndImportBlockAtSlot(final long slot) throws
     return createAndImportBlockAtSlot(UnsignedLong.valueOf(slot));
   }
 
+  public BlockProcessingRecord createAndImportBlockAtSlot(
+      final UnsignedLong slot, List<Attestation> attestations) throws Exception {
+    SSZList<Attestation> sszList =
+        attestations.isEmpty()
+            ? null
+            : new SSZList<>(attestations, Constants.MAX_ATTESTATIONS, Attestation.class);
+
+    return createAndImportBlockAtSlot(slot, Optional.ofNullable(sszList));
```

Comment:
Maybe just:
```
Optional<SSZList<Attestation> sszList = attestations.isEmpty() ? Optional.empty() : Optional.of(new SSZList<>(...))
```

to save mixing `null` and `Optional`.

Reply:
Done